tests/console/yast2_clone_system.pm

```diff
@@ -42,7 +42,7 @@ sub run {
     # Check and upload profile for chained tests
     upload_asset $ay_profile_path;
 
-    unless (is_opensuse) {
+    unless (is_opensuse || is_staging) {
```

Comment:
Not sure if that's correct. It's same as `if (!is_opensuse && !is_staging). It means, that for staging this will be false, which I think is wrong. Am I missing something?